### PR TITLE
Migrate our documentation to mkdocs

### PR DESCRIPTION
**Pull request by pappacena**
Migrate our documentation to mkdocs to:
- Have a better look and feel (using material theme)
image

- Better display of code snippets (including interactive line explanation buttons)
image

- Have a short quick start guide going from installation to prediction with few small snippets
- Change the documentation structure to separate in sections (mainly, "image acquisition", "running prediction" and "image operations (to be created)")
image

### CODE REVIEW

File: docs/image-acquisition/image-file.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+from landingai.pipeline.frameset import Frame
+
+frame = Frame.from_image("/path/to/your/image.jpg")
+frame.resize(width=512, height=512)
+frame.save_image("/tmp/resized-image.png")
+```
+
+1. Open `/path/to/your/image.jpg` image file.
+2. Resize the frame to `width=512px` (keeping aspect ratio)
+3. Save the resized image to `/tmp/resized-image-<index>.png`.
```
Comment (Dayof) [inline docs/image-acquisition/image-file.md:12]: `/tmp/resized-image.png`.

File: docs/image-acquisition/image-file.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+Creating a frame out of an image file is pretty straightforward:
+```py
+from landingai.pipeline.frameset import Frame
+
+frame = Frame.from_image("/path/to/your/image.jpg")
+frame.resize(width=512, height=512)
+frame.save_image("/tmp/resized-image.png")
```
Comment (Dayof) [inline docs/image-acquisition/image-file.md:7]: here is missing the tooltips numbers right ?

File: docs/image-acquisition/network-cameras.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+```py
+from landingai.pipeline.image_source import NetworkedCamera
+from landingai.pipeline.frameset import FrameSet
+
+frameset = FrameSet() # (1)!
+with NetworkedCamera(stream_url="rtsp://192.168.0.77:8080/h264_opus.sdp", fps=1) as camera: # (2)!
+    for i, frame in enumerate(camera): # (3)!
+        if i >= 100:
+            break
+        frame.resize(width=256) # (4)!
```
Comment (Dayof) [inline docs/image-acquisition/network-cameras.md:15]: if previously you were applying frame.resize(width=512, height=512) to keep aspect ratio, why now you only resize to the width to 256?

File: docs/image-acquisition/webcam.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
+Just like [video files](image-acquisition/video-file.md), you can extract frames from your local webcam using `landingai.image_source.Webcam`.
```
Comment (Dayof) [inline docs/image-acquisition/webcam.md:1]: landingai.pipeline.image_source.Webcam

File: docs/index.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+with Webcam(fps=0.5) as webcam:  # (1)!
+    for frame in webcam:
+        frame.resize(width=512, height=512) # (2)!
+        frame.save_image("/tmp/webcam-image.png") # (3)!
+```
+
+1. Capture images from the webcam at 0.5 frames per second (1 frame every 2 seconds), closing the camera at the end of the `with` block
+2. Resize the frame to 512x512 pixels
+3. Save the images as `/tmp/webcam-image-<timestamp>.png`
```
Comment (Dayof) [inline docs/index.md:53]: this <timestamp> is a suffix we always add?
if so we need to add this in the other snippets doc

File: docs/index.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+        frame.overlay_predictions()
+        if "coffee-mug" in frame.predictions:  # (6)!
+            frame.save_image("/tmp/latest-webcam-image.png", include_predictions=True) # (7)!
+```
+
+1. Creates a LandingLens predictor object
+2. Here, you should set the endpoint_id you've got from the platform: <br/>![How to get endpoint ID](images/copy-endpoint-id.png "How to get endpoint ID").
+3. Here, you should set the API key you've got from the platform: <br/> ![How to get the API key](images/menu-api-key.png "How to get the API key").
+4. Resize the image to `width=512`, keeping the aspect ratio. This is useful to save some bandwidth when sending the image to LandingLens for prediction.
+5. Runs inference in the resized frame, and returns a new frame with the predictions
```
Comment (Dayof) [inline docs/index.md:91]: we are not capturing this frame returned, so I imagined that it's not actually return but more like an internal diff state right?

File: docs/inferences/frames-inference.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
+Now that you have the API key and an endpoint ID (if you don't, see [Running inferences / Getting started](inferences/getting-started.md) section), you can run your first inference using the SDK.
+
+## Working with frames
+
+The SDK provides a variety of image sources helpers that can be used to acquire images (see [Image acquisition](image-acquisition/image-acquisition.md) section), and all those image sources yields `Frame` objects, that can be used to run inferences.
+
+The process is prety straightforward. To run inferences in frames extracted from the webcam, for example, you will just use the `Webcam` and the `Predictor` classes:
```
Comment (Dayof) [inline docs/inferences/frames-inference.md:7]: pretty

File: docs/inferences/frames-inference.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+        frame.run_predict(predictor=predictor)
+        if "coffee-mug" in frame.predictions.filter_threshold(0.95):
+            print(f"I'm super sure I found a coffee-mug in the frame!")
+```
+
+## Predictor rate limit
+
+The `Predictor` object is the main object used to run inferences. It calls LandingLens API to execute inferences using the hosted model.
+
+Keep in mind that LandingLens has rating limit that veryes with your plan. At the moment, non-enterprise plans allows for up to 40 requests per minute. If you exceed this limit, your Predictor object will retry the request automatically after some seconds.
```
Comment (Dayof) [inline docs/inferences/frames-inference.md:62]: varies

File: docs/inferences/frames-inference.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+
+Please, adjust your image capturing rate to avoid exceeding the limit, or contact LandingAI team ([sales@landing.ai](sales@landing.ai)) to upgrade your plan for higher limits or local inferences support.
```
Comment (Dayof) [inline docs/inferences/frames-inference.md:64]: nice 👍🏼

File: docs/inferences/getting-started.md [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+To run inferences using LandingLens, you must first build a model. If you didn't sign up before, visit https://app.landing.ai/, sign up for a free account and create a new project. If you are not familiar with LandingLens, you can find a lot of useful information in the [LandingLens support center](https://support.landing.ai/docs/landinglens-workflow).
+
+Long story short, after creating a project in LandingLens you will need to:
+
+1. Upload example images
+2. Label those images
+3. Train a model by clicking the "Train" button
+4. Once the model is trained, you will need to click the "Deploy" button to deploy the model to an endpoint provided by LandingLens
+
+The image below ilustrate LandingLens painel with some example images already labeled, and the "Train" and "Deploy" buttons.
```
Comment (Dayof) [inline docs/inferences/getting-started.md:15]: panel

File: docs/inferences/getting-started.md [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+Once you are ready to [acquire images](image-acquisition/image-acquisition.md), it's now time to perform inferences on them. This section will cover the basics of how to perform inferences on images to classify, detect objects or create segmentation masks on the acquired frames.
+
+## Building your first model
+
+To run inferences using LandingLens, you must first build a model. If you didn't sign up before, visit https://app.landing.ai/, sign up for a free account and create a new project. If you are not familiar with LandingLens, you can find a lot of useful information in the [LandingLens support center](https://support.landing.ai/docs/landinglens-workflow).
+
+Long story short, after creating a project in LandingLens you will need to:
+
+1. Upload example images
+2. Label those images
```
Comment (Dayof) [inline docs/inferences/getting-started.md:11]: the example images are already label right?

File: docs/image-acquisition/image-file.md (hunk 2 above)
Comment (pappacena) [reply docs/image-acquisition/image-file.md:7]: Good catch!

File: docs/image-acquisition/network-cameras.md (hunk 3 above)
Comment (pappacena) [reply docs/image-acquisition/network-cameras.md:15]: The previous comment was wrong. It was not keeping aspect ratio.

File: docs/index.md (hunk 5 above)
Comment (pappacena) [reply docs/index.md:53]: Nope. The previously merged PR fixes this, but I forgot to fix in the docs. Changing it now.

File: docs/index.md (hunk 6 above)
Comment (pappacena) [reply docs/index.md:91]: Yes! Let me change the copy.

File: docs/inferences/getting-started.md (hunk 11 above)
Comment (pappacena) [reply docs/inferences/getting-started.md:11]: Hum... I think the term "examples" might be misleading, confusing with the dataset samples we have in the platform. Let me change that a bit.

**Pull request merged**